install/Knative-with-GiantSwarm.md

```diff
@@ -0,0 +1,184 @@
+# Knative Install on Giant Swarm cluster
```

Comment:
You should also edit the /install/README.md file to include a link to this install file.